common/scripted_effects/00_childhood_effects.txt

```diff
@@ -544,8 +544,8 @@ save_good_character_effect = {
 				}
 				modifier = {
 					add = 4
-					age >= root.age_dif_child_down
-					age <=  root.age_dif_child_up
+					age > root.age_dif_child_down
+					age < root.age_dif_child_down
```

Comment:
I don't geti. It's equal to `age = root.age_dif_child_down` and that's not what was in the vanilla.

Reply:
Me neither, don't quite know what I was doing there... Feel free to change it back to vanilla.

Reply:
Bring back the vanilla, yes.